src/gutter-component.coffee

```diff
@@ -238,3 +251,9 @@ GutterComponent = React.createClass
         editor.unfoldBufferRow(bufferRow)
       else
         editor.foldBufferRow(bufferRow)
+
+  show: ->
```

Comment:
Same here, I think show and hide are methods on the model and the component just updates to reflect its state. Very non-traditional React we want the user-facing API to be a plain-old-JavaScript-object so the methods should be there.